redbot/cogs/downloader/downloader.py

```diff
@@ -792,63 +808,66 @@ def __init__(self, bot: Red):
                         )
                     for page in pagify(msg):
                         await ctx.send(msg)
-                    return
+
                 except errors.UnknownRevision:
-                    await ctx.send(
-                        _("Error: there is no revision `{rev}` in repo `{repo.name}`").format(
-                            rev=rev, repo=repo
-                        )
-                    )
-                    return
-                await repo.checkout(commit)
-                cogs_to_check = await self._get_cogs_to_check(repos=[repo], cogs=cogs)
+                    message += _(
+                        "Error: there is no revision `{rev}` in repo `{repo.name}`"
+                    ).format(rev=rev, repo=repo)
+
```

Comment:
Same as above, the `return` shouldn't be removed + the message has to still be sent.